admin/app/services/access-control.js

```diff
@@ -25,6 +25,10 @@ export default class AccessControlService extends Service {
     );
   }
 
+  get hasAccessToTrainingsCreationActionsScope() {
+    return !!(this.currentUser.adminMember.isSuperAdmin || this.currentUser.adminMember.isMetier);
```

Comment:
Je ne pense pas que ce soit nécessaire

```suggestion
    return this.currentUser.adminMember.isSuperAdmin || this.currentUser.adminMember.isMetier;
```